src/Searchbar/Searchbar.scss

```diff
@@ -5,3 +5,4 @@
 .ant-select-item-option-content {
   white-space: normal;
 }
+
```

Comment:
👍  normally I'd say this file should not have been touched since it was not part of your changes though. Small styling fixes like these are great, but better to be left in their own "cleanup" PRs